teps/0069-support-retries-for-custom-task-in-a-pipeline.md

```diff
@@ -126,7 +126,7 @@ Proposed algorithm for performing a retry for custom task.
   to request a custom task to cancel.
   
 - Step 3. In addition to patching the `pipelinerun` controller also enqueue a timer
-  `EnqueueAfter(30*time.Second)` (configurable). On completion of timeout
+  `time.After(30*time.Second)` (configurable). On completion of timeout
```

Comment:
using `time.After` for now.

Once the knative pkg is updated, I will be able to switch to `NewRequeueAfter`.

Reply:
can you explain a bit more about this change? (maybe some links to docs e.g. about `NewRequeueAfter`) is there are reason we couldn't continue to use `EnqueueKeyAfter` in the meantime (i.e. is there something wrong with it?) also looking at https://github.com/tektoncd/pipeline/pull/4131 it looks like `NewRequeueAfter` is available now